reference/filesystem/functions/file.xml

```diff
@@ -1,76 +1,205 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!-- EN-Revision: n/a Maintainer: fernando Status: working -->
-  <!-- CREDITS: cortesi -->
-  <refentry xml:id="function.file" xmlns="http://docbook.org/ns/docbook">
-   <refnamediv>
-    <refname>file</refname> 
-    <refpurpose>Legge l'intero file in un vettore</refpurpose>
-   </refnamediv>
-   <refsect1>
-    <title>Descrizione</title>
-     <methodsynopsis>
-      <type>array</type><methodname>file</methodname>
-      <methodparam><type>string</type><parameter>filename</parameter></methodparam>
-      <methodparam choice="opt"><type>int</type><parameter>use_include_path</parameter></methodparam>
-      <methodparam choice="opt"><type>resource</type><parameter>context</parameter></methodparam>
-     </methodsynopsis>
-    <para> 
-     Identica a <function>readfile</function>, eccetto per il fatto che
-     <function>file</function> restituisce il file in un vettore. Ogni
-     elemento del vettore corrisponde ad una riga del file, con il carattere di
-     newline ancora inserito. Se la funzione non riesce restituisce 
-     &false;.
-    </para>
-    <para>
-     Puoi impostare il secondo parametro, <parameter>use_include_path</parameter>, (opzionale) ad "1",
-     se vuoi cercare il file nel <link
-     linkend="ini.include-path">include_path</link>.
-    </para>
-    <para>
-     <informalexample>
-      <programlisting role="php">
+<?xml version="1.0" encoding="utf-8"?>
+<!-- EN-Revision: e3ae86b439c1963f0cd6d262712e2d3f7e85bc37 Maintainer: ManueldG Status: working -->
+<!-- CREDITS: cortesi-->
+<refentry xmlns="http://docbook.org/ns/docbook" xml:id="function.file">
+ <refnamediv>
+  <refname>file</refname>
+  <refpurpose>Legge l'intero file in un array</refpurpose>
+ </refnamediv>
+ 
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis>
+   <type class="union"><type>array</type><type>false</type></type><methodname>file</methodname>
+   <methodparam><type>string</type><parameter>filename</parameter></methodparam>
+   <methodparam choice="opt"><type>int</type><parameter>flags</parameter><initializer>0</initializer></methodparam>
+   <methodparam choice="opt"><type class="union"><type>resource</type><type>null</type></type><parameter>context</parameter><initializer>&null;</initializer></methodparam>
+  </methodsynopsis>
+  <para>
+   Legge l'intero file in un array.
+  </para>
+  <note>
+   <para>
+    In alternativa puoi usare <function>file_get_contents</function> per estrarre il contenuto
+    di un file sottoforma di stringa.
```

Comment:
[Usa "sotto forma" in alternativa di "sottoforma". ](https://www.treccani.it/magazine/lingua_italiana/domande_e_risposte/grammatica/grammatica_290.html)